README.md

```diff
@@ -29,15 +29,15 @@ Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps gi
 
 3.  Create a symlink of the repository in the global folder
```

Comment:
I liked your version of the sentence here, i.e., "Add a symbolic link to the global folder" . I meant to restore the second point only. I'll be happy to have your version of the statement here.